rust/main/Cargo.toml

```
@@ -99,8 +76,8 @@ hyper = "0.14"
 hyper-tls = "0.5.0"
 hyperlane-cosmwasm-interface = "=0.0.6-rc6"
 injective-protobuf = "0.2.2"
-injective-std = "0.1.5"
-itertools = "*"
+injective-std = "=0.1.5"
+itertools = "0.12.0"
```

keeping as is doesn't break the build, would prefer this
```suggestion
itertools = "*"
```

---

Unpinned ✅